commands/create_kubeconfig.go

```diff
@@ -452,43 +407,28 @@ func createKubeconfig(args createKubeconfigArguments) (createKubeconfigResult, e
 		}
 	} else {
 		// create a self-contained kubeconfig
-		kubeconfig := Kubeconfig{
-			APIVersion:     "v1",
-			Kind:           "Config",
-			CurrentContext: args.contextName,
-			Clusters: []KubeconfigNamedCluster{
-				{
-					Name: "giantswarm-" + args.clusterID,
-					Cluster: KubeconfigCluster{
-						Server:                   result.apiEndpoint,
-						CertificateAuthorityData: base64.StdEncoding.EncodeToString([]byte(response.Payload.CertificateAuthorityData)),
-					},
-				},
-			},
-			Contexts: []KubeconfigNamedContext{
-				{
-					Name: args.contextName,
-					Context: KubeconfigContext{
-						Cluster: "giantswarm-" + args.clusterID,
-						User:    "giantswarm-" + args.clusterID + "-user",
-					},
-				},
-			},
-			Users: []KubeconfigUser{
-				{
-					Name: "giantswarm-" + args.clusterID + "-user",
-					User: KubeconfigUserKeyPair{
-						ClientCertificateData: base64.StdEncoding.EncodeToString([]byte(response.Payload.ClientCertificateData)),
-						ClientKeyData:         base64.StdEncoding.EncodeToString([]byte(response.Payload.ClientKeyData)),
-					},
+		var yamlBytes []byte
+		{
+			c := k8srestconfig.Config{
+				Logger: microloggertest.New(),
+
+				Address:   result.apiEndpoint,
+				InCluster: false,
+				TLS: k8srestconfig.TLSClientConfig{
```

Comment:
```suggestion
				TLS: k8srestconfig.ConfigTLS{
```

This got renamed. You'll need to do a vendor update for operatorkit.

Reply:
Thanks, didn't know that

Reply:
I posted it in #news-dev because it was a breaking change albeit a small one. 

https://gigantic.slack.com/archives/C04TGHDEF/p1552555043000200

It's good to keep up to date with the news channels for stuff like this.